src/installer/managed/Microsoft.NET.HostModel/ResourceUpdater.cs

```diff
@@ -225,11 +41,34 @@ public static bool IsSupportedOS()
         /// updates.
         /// </summary>
         public ResourceUpdater(string peFile)
+            : this(new FileStream(peFile, FileMode.Open, FileAccess.ReadWrite, FileShare.None))
+        {
+        }
+
+        /// <summary>
+        /// Create a resource updater for the given PE file. This will
+        /// acquire a native resource update handle for the file,
```

Comment:
I'm curious about the "native resource update handle" - I don't think that applies anymore. The overall meaning of the comment still holds, but this specific doesn't seem to be true anymore.